acoular/sdinput.py

```diff
@@ -49,6 +50,11 @@ def __init__(self, *args, **kwargs):
     #: Sampling frequency of the signal, changes with sinusdevices
     sample_freq = Property(desc='sampling frequency')
 
+    _sample_freq = Float(0)
+
+    #: Datatype (resolution) of the signal, used as `dtype` in a sd `Stream` object
+    data_type = Enum([float32, float16, int32, int16, int8, uint8], desc='data type (resolution) of signal')
```

Comment:
To be consistent with other classes in Acoular, this trait should be named `precision` and the type should be `str`. The sounddevice `InputStream` object accepts a string for the `dtype` argument 

Thus: `precision = Trait('float32', 'float16',...)`